conda/recipes/libcudf_kafka/meta.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2018, NVIDIA CORPORATION.
+
+{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') + environ.get('VERSION_SUFFIX', '') %}
+{% set minor_version =  version.split('.')[0] + '.' + version.split('.')[1] %}
+
+package:
+  name: libcudf_kafka
+  version: {{ version }}
+
+source:
+  git_url: ../../..
+
+build:
+  number: {{ GIT_DESCRIBE_NUMBER }}
+  string: {{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
+  script_env:
+    - CC
+    - CXX
+    - CUDAHOSTCXX
+    - PARALLEL_LEVEL
+    - VERSION_SUFFIX
+
+requirements:
+  build:
+    - cmake >=3.14.0
+  host:
+    - libcudf {{ version }}
+    - librdkafka=1.4.0
+  run:
+    - librdkafka=1.4.0 #TODO: librdkafka should be automatically included here by run_exports but is not
```

Comment:
cc @jakirkham we had to add `librdkafka` to the `run` section here otherwise it wouldn't properly enforce a runtime dependency

Reply:
It seems there is no space between the library and the version spec, which is something conda-build expects. Not sure if that is the cause or just one issue.

Reply:
It previously had the space and we tried both with having the `run:` and without it and neither seemed to enforce the run export.

Reply:
Yeah just pointing out that either way this is a syntax error. Though maybe conda-build sort of gets past it somehow.